### PR TITLE
Added support of iDEAL to the Instant Component.

### PR DESCRIPTION
**Pull request by Robert-SD**

### CODE REVIEW

File: android/src/main/kotlin/com/adyen/checkout/flutter/components/instant/InstantComponentManager.kt [hunk 1]
```diff
@@ -132,7 +188,58 @@ class InstantComponentManager(
...
+            paymentMethod = paymentMethod,
+            checkoutConfiguration = configuration,
+            componentCallback =
+                IdealComponentSessionCallback(
+                    componentFlutterInterface,
+                    componentId,
+                    ::handleAction,
+                    ::hideLoadingBottomSheet
+                ),
+            key = UUID.randomUUID().toString()
```
Comment (ozgur00) [inline android/src/main/kotlin/com/adyen/checkout/flutter/components/instant/InstantComponentManager.kt:237]: Just checking: Are you intentionally attempting to create a new instance every time?

Comment (Robert-SD) [reply android/src/main/kotlin/com/adyen/checkout/flutter/components/instant/InstantComponentManager.kt:237]: Yes, this is intentionally to support multiple simulations components. Would you suggest to use only one instance?

Comment (ozgur00) [reply android/src/main/kotlin/com/adyen/checkout/flutter/components/instant/InstantComponentManager.kt:237]: Not necessarily. Unless merchants would want to access to the same Ideal Component which is not possible at this point. At this point if this aligns with iOS behavior, it's perfectly fine. I would only recommend to consider if in the future flutter sdk would support this or not.

Comment (Robert-SD) [reply android/src/main/kotlin/com/adyen/checkout/flutter/components/instant/InstantComponentManager.kt:237]: Alright, I would suggest to keep it unique for now. Whenever we might get the demand to reuse or access the instant component, we could offer it. But for now I think its fine.

**Review by ozgur00 — APPROVED** [review]

**Pull request merged**